AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -171,6 +171,8 @@ eatsmarter.de,inside-digital.de,hs.fi,vrt.be,manager-magazin.de,whats-on-netflix
 !
 !
 !
+maniac.de#$#body > div[id*="-"][data-bg^="background-color:"][style*="backdrop-filter:"] { display: none !important; }
+maniac.de#$#body { overflow: auto !important; }
```

Comment:
I suggest to change the rule a bit due to the cursor.

```
maniac.de#$#body { overflow: auto !important; cursor: auto !important; }
```

Reply:
Do you want to move these rules to AG German filter with the following changes?
```suggestion
maniac.de#$#body { overflow: auto !important; }
```